saleor/settings.py

```diff
@@ -542,3 +566,10 @@ def get_host():
 
 SERIALIZATION_MODULES = {
     'json': 'saleor.core.utils.json_serializer'}
+
+
+# you can override settings also in local_settings.py
+try:
+    from settings_local import *  # noqa
```

Comment:
You should use `DJANGO_SETTINGS_MODULE` and point it to your `settings_local`, instead of importing `settings_local` to the normal `settings` file
https://docs.djangoproject.com/en/2.1/topics/settings/#envvar-DJANGO_SETTINGS_MODULE

Reply:
This not same. Django documentation say what you can define Django settings file, but don't say how apply settings on already forked project with minimal changing. I try fix this situation.
Lets see:
settings.py - big, around 550 lines! Copy and full edit - bad way. My way - allow don't do any changing in original settings.py file, just define variables from os.env or if you wish from .env file. In any way no conflicts. If you still need change  some settings you can too use settings_local.py file (example for define not trivial - Logging settings).

I don't say what my solution perfect, but Saleor don't have any solution for small pain update core code. Can explain how you company work with this problem?

Reply:
1. my solution give you choise: os.env and/or .env file, solution - full compatible, this not replacement! What edit more simple os variables or file? Where we can search any string on os env or in files??? etc... I think simple - use .env file, you already use it, see common.env))) Just try on clear machine clone Saleor with my PR and without and compare how much time you lost just for run Saleor with default configuration. I love just do clone to favorite IDE and simple run code, i don't wont do what must work from box. If this don't work from box - maybe this don't general work? I remember when first time clone Saleor - and i have simple questions: why i must set secret_key, i just wont run this, why developers dont set any default secret key? I just wont run, i select from oscar, odoo etc.... i need simple way to run!...What, how in 2018 year shop-framework dont have demo docker??? Next shock - what tab release is not release)))) I see all time questions like this on gitter-chat, almost every day.
2. why what you say don't present in docs in "how customize code"? Many peoples just see files and this use how docs. Have one good old rule - good software - don't need help docs. 
3. Saleor - great shop, but must be more fork-based framework friendly from box! I think...

Reply:
@mmmcorpsvit Please follow my line of though for a second:

1. Create a file named `local_settings.py`

2. Have the contents say:

    ```python
    from saleor.settings import *

    TEMPLATES[0]['DIRS'].append('local_templates')
    ```

3. Launch Django with `--settings-module=local_settings`

Everything should work with none of the proposed changes applied.

As for `local_settings` vs. `settings_local`: is it better to have local settings closer to global settings and local templates closer to global templates or is it better to have local overrides close to each other (ie. have local templates next to local settings)?